descartes_moveit/include/descartes_moveit/moveit_state_adapter.h

```diff
@@ -85,7 +86,16 @@ class MoveitStateAdapter : public descartes_core::RobotModel
    * @brief Pointer to moveit robot state (mutable object state is reset with
    * each function call
    */
+
+  /**
+   * TODO: Checks for collisions at this joint pose. The setCollisionCheck(true) must have been
```

Comment:
This seems counter-intuitive.  I can see using the flag to turn on/off collision checking for IK, but anybody calling this method should expect it to perform collision checking.